crates/solver/src/settlement_post_processing/mod.rs

```diff
@@ -143,7 +143,14 @@ impl PostProcessing for PostProcessingPipeline {
                     prices,
                     &solver_account.address(),
                 )
-                .await,
+                .await
+                .map_or_else(
+                    |err| {
+                        tracing::warn!("Failed to compute score: {}", err);
```

Comment:
Usually this results in nicer logging.
```suggestion
                        tracing::warn!(?err, "failed to compute score");
```

Reply:
fixed